bin/lesson_check.py

```diff
@@ -388,10 +388,6 @@ def get_loc(self, node):
 class CheckNonJekyll(CheckBase):
     """Check a file that isn't translated by Jekyll."""
 
-    def __init__(self, args, filename, metadata, metadata_len, text, lines, doc):
-        super(CheckNonJekyll, self).__init__(args, filename, metadata, metadata_len, text, lines, doc)
-
-
```

Comment:
I always include explicit constructors to make it clear to novices where to add new initialization code.